telethon/network/connection/tcpmtproxy.py

```diff
@@ -130,6 +132,21 @@ def address_info(proxy_info):
             raise ValueError("No proxy info specified for MTProxy connection")
         return proxy_info[:2]
 
+    @staticmethod
+    def normilize_secret(secret):
+        if secret[:2] in ["ee","dd"]: # Remove extra bytes
```

Comment:
No need for a mutable list (and formatting, space after comma):

```suggestion
        if secret[:2] in ("ee", "dd"):  # Remove extra bytes
```